Library/ViewModels/CommentCellViewModel.swift

```diff
@@ -85,13 +85,32 @@ public final class CommentCellViewModel:
       .map { comment, _ in comment }
 
     self.authorImageURL = comment
-      .map(\.author.imageUrl)
+      .map {
+        if $0.author.isBlocked {
+          return "" // Use default avatar.
```

Comment:
Will this default avatar URL. be hardcoded somewhere? Or is some other piece of code checking for this empty string?

You could also throw this in the model code, instead. It's a little sneaky but means anyone working with/implementing comment code in the future doesn't need to remember this edge case.

Reply:
I think we may want to replace the current gray circle with an icon instead, but for now empty string -> nil url -> gray placeholder circle. I added the comment since I didn't feel `""` was clear about that by itself. Glad to see I was right about that! Would it be more clear if I said "Keep placeholder avatar" instead?

Reply:
Yeah, maybe something like `// Use placeholder avatar instead of URL`?